community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/planDescription/InternalPlanDescription.scala

```diff
@@ -91,6 +93,16 @@ object InternalPlanDescription {
     case class KeyExpressions(expressions: Seq[commands.expressions.Expression]) extends Argument
     case class EntityByIdRhs(value: PipeEntityByIdRhs) extends Argument
     case class EstimatedRows(value: Double) extends Argument
+    case class Signature(procedureName: QualifiedProcedureName,
```

Comment:
You are breaking symmetry here - the other Argument classes do not know how to produce text. That power is instead with PlanDescriptionArgumentSerializer. I don't mind either way, but it should probably be consistent